apps/backend/alembic/versions/f4a2b1c8d37e_create_unified_infrastructure_schema.py

```
@@ -0,0 +1,337 @@
+    op.drop_index('idx_devices_hostname', table_name='devices')
+    
+    # Drop tables
+    op.drop_table('cache_metadata')
+    op.drop_table('service_performance_metrics')
+    op.drop_table('configuration_change_events')
+    op.drop_table('configuration_snapshots')
+    op.drop_table('data_collection_audit')
+    op.drop_table('drive_health')
+    op.drop_table('container_snapshots')
+    op.drop_table('system_metrics')
+    op.drop_table('devices')
```

Ruff (0.12.2)

No newline at end of file

Add trailing newline

(W292)